### PR TITLE
Mitra/Reusable hero banner

### PR DESCRIPTION
**Pull request by mitra-deriv**
Changes:

Co-authored-by: @farzin-deriv

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by farzin-deriv — COMMENTED** [review]
@mitra-deriv Just a suggestion, instead of `derivez-logo.svg`, It's better to have the icon and the text logo as separate files, So later in the code, if needed we can use them separately or put them together with the code, So we won’t need to add a new asset again in the future if we need the logo.

File: src/pages/derivez/index.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+`
+
+const Hero = () => {
+    const { is_mobile } = useBreakpoints()
+    return (
+        <BackgroundStyle>
+            <ContentWrapperStyle>
+                <Content>
+                    <img width="237px" height="64px" src={DerivEZLogo} />
```
[the remark this answers is not in the document]
Comment (mitra-deriv) [reply src/pages/derivez/index.tsx:75]: @farzin-deriv Thanks for the time and Reviewing the PR , I will resolve them in another branch and mark the conversations as resolved once it is done 🙏 🌹 

About the platforms logos just to be sure I got your point correctly,  You mean we create a component which accepts  the `icon` (the red one)  +  `text` (which here is deriv ez ) ? 


Screenshot 2023-02-07 at 9 44 15 AM

[the remark this answers is not in the document]
Comment (farzin-deriv) [reply src/pages/derivez/index.tsx:75]: Not accepting icon and text as props, The component will handle it internally. Let's have a talk when you have time so I can explain it 🙇🏻

**Pull request merged**